script/reviewer.go

```diff
@@ -19,11 +20,84 @@ func (r *Reviewer) Review() {
 
 	r.application.Parse(r.gitHub.Issue)
 
-	if isTestingIssue() {
-		if r.application.IsValid() {
-			debugMessage("Application has no problems")
-		} else {
-			debugMessage("Application problems:", r.application.RenderProblems())
+	status := r.getStatus()
+	r.updateLabels(status)
+	r.createComment(status)
+}
+
+func (r *Reviewer) getStatus() string {
+	if *r.gitHub.Issue.State == "closed" {
+		return "closed"
+	} else if r.gitHub.IssueHasLabel(LabelStatusApproved) {
+		return "approved"
+	} else if r.gitHub.IssueHasLabel(LabelStatusReviewing) {
+		return "reviewing"
+	} else if r.gitHub.IssueHasLabel(LabelStatusInvalid) {
+		return "invalid"
+	} else {
+		return "new"
+	}
+}
+
+func (r *Reviewer) createComment(status string) {
+	title := ""
+	body := ""
+	details := fmt.Sprintf("<details>\n<summary>Application data...</summary>\n\n```json\n%s\n```\n</details>", r.application.GetData())
+
+	if status == "closed" {
+		body = "Oops! This application is closed can no longer be processed. If this is an error, please reach out to [opensource@1password.com](mailto:opensource@1password.com)."
+	} else if status == "approved" {
+		body = "Oops! This application has been updated but has already been approved and can no longer be processed. If this is an error, please reach out to [opensource@1password.com](mailto:opensource@1password.com)."
+	} else if status == "reviewing" && r.application.IsValid() {
+		title = "### 👍 Application still valid"
+		body = fmt.Sprintf("\n\n%s\n\nWe've processed your updated application and everything still looks good.", details)
```

Comment:
1. Similar to the previous statement, I don't actually know what this means for the user. What does "processed" mean? What does "everything still looks good" mean?
2. We use curly apostrophes <kbd>Opt</kbd>-<kbd>Shift</kbd>-<kbd>]</kbd> instead of straight apostrophes, per [the Style Guide](https://support.apple.com/en-gb/guide/applestyleguide/apsg3acde405/1.0/web/1.0#apd1d1eb8bec58f4).

Reply:
> What does "processed" mean?

Tough one - basically this is the "automated review" stage that occurs, but there is also a human that comes and does a final check/review. So I'm hesitant to say "We've reviewed your updated application".

Maybe something like this:

> We've evaluated your updated application and it is still valid.

Reply:
Love it!